src/menu_items.c

```diff
@@ -5702,7 +5732,7 @@ void add_menu_item(s32 type, s32 column, s32 row, s8 priority) {
         case MAIN_MENU_BACKGROUND:
         case CHARACTER_SELECT_BACKGROUND:
         case COURSE_SELECT_BACKGROUND:
-            load_texture_reset_cache(gMenuTexturesBackground[has_unlocked_extra_mode()]->textureData);
+            load_menu_img_comp_type(gMenuTexturesBackground[has_unlocked_extra_mode()]->textureData, LOAD_MENU_IMG_TKMK00_ONCE);
```

Comment:
This might break the greyscale backgrounds

Reply:
Oh my bad was trying on background try multiple thing but forget to revert